gridded_etl_tools/utils/store.py

```diff
@@ -153,7 +151,7 @@ def path(self) -> str:
             A URL string starting with "s3://" followed by the path to the Zarr.
         """
 
-    def dataset(self, **kwargs: dict) -> xr.Dataset | None:
+    def dataset(self) -> xr.Dataset | None:
```

Comment:
The kwargs provides a shortcut to getting a refreshed mapper and therefore non-cached dataset but a quick grep shows it should be safe to remove if desired. If so, the docstring needs updating.

Reply:
This was an artifact of an earlier draft where I got rid of `**kwargs` from `mapper`, and then realized they were being used. I should add `**kwargs` back here, too. I would eventually like to see them phased out, but in a deliberate, thoughtful, and tested way.

Reply:
62db238